alchemy-web/docs/providers/sentry/client-key.md

```diff
@@ -31,6 +31,65 @@ Get your auth token from [Sentry's API settings](https://sentry.io/settings/acco
 
 ## Usage
 
+```typescript
+import { ClientKey } from "@alchemy/providers/sentry";
+
+// Create a basic client key
+const key = await ClientKey("my-key", {
+  name: "My Key",
+  project: "my-project",
+  organization: "my-org"
+});
+
+// Create a client key with rate limiting
+const key = await ClientKey("rate-limited-key", {
+  name: "Rate Limited Key",
+  project: "my-project",
+  organization: "my-org",
+  rateLimit: {
+    window: 3600, // 1 hour
+    count: 1000   // 1000 events per hour
+  }
+});
+
+// Create a client key with a custom auth token
+const key = await ClientKey("my-key", {
+  name: "My Key",
+  project: "my-project",
+  organization: "my-org",
+  authToken: alchemy.secret("my-auth-token")
+});
+```
+
+## Properties
+
+| Property | Type | Required | Description |
+|----------|------|----------|-------------|
+| `name` | `string` | No | The name of the key |
+| `project` | `string` | Yes | The project slug that owns the key |
+| `organization` | `string` | Yes | The organization ID or slug that owns the key |
+| `rateLimit` | `object` | No | Rate limit configuration |
+| `useCase` | `string` | No | The use case for the key |
+| `authToken` | `Secret` | No | Auth token to use (overrides environment variable) |
+| `adopt` | `boolean` | No | Whether to adopt an existing key with the same name if it exists |
+
+## Output
+
+| Property | Type | Description |
+|----------|------|-------------|
+| `id` | `string` | The ID of the key |
+| `name` | `string` | The name of the key |
+| `label` | `string` | The label of the key |
+| `public` | `string` | The public key |
+| `secret` | `string` | The secret key |
+| `projectId` | `number` | The project ID |
+| `isActive` | `boolean` | Whether the key is active |
+| `dsn` | `object` | DSN configuration |
+| `browserSdkVersion` | `string` | Browser SDK version |
+| `browserSdk` | `object` | Browser SDK choices |
+| `dateCreated` | `string` | Time at which the key was created |
+| `dynamicSdkLoaderOptions` | `object` | Dynamic SDK loader options |
```

Comment:
nit: Haven't been including this in the docs with the other resources. Probably something we'll standardize when @FutureExcited generates the docs from the source code.

Reply:
Resolved via 4f8853d082d163233a6800906ce39917e4981653